gapis/trace/android/profile/groups.go

```diff
@@ -79,6 +80,30 @@ func (t *GroupTree) CreateOrGetGroup(name string, link sync.SubCmdRange) int32 {
 	return rp.id
 }
 
+// CreateOrGetDrawCallGroup finds or creates the group for the given draw call command and returns its id.
+// TODO: the same assumptions apply here as do for CreateOrGetGroup.
+func (t *GroupTree) CreateOrGetDrawCallGroup(name string, link api.SubCmdIdx) int32 {
```

Comment:
It's a nit, but I think the standard nomenclature is "getOrCreateX" rather than "createOrGetX()"

Reply:
Done.